pro-plan-for-free.js

```diff
@@ -6,5 +6,6 @@
 
 module.exports = [
   'resistbot', // 2018-10-23: Volunteer run org, helping people engage in their democracy
-  'itspugle' // 2018-10-24: Student creating tools to better enable volunteer groups
+  'itspugle' // 2018-10-24: Student creating tools to better enable volunteer groups,
```

Comment:
you’ll need to add a `,` or it will break the app :)

```suggestion
  'itspugle', // 2018-10-24: Student creating tools to better enable volunteer groups,
```

Reply:
Done, Sorry about that.